plonk-core/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "plonk-core"
-version = "0.8.2"
+version = "0.0.0"
```

```suggestion
version = "0.1.0"
```

Minimal version required. Can't be 0.0.0